test/functional/treesitter/parser_spec.lua

```diff
@@ -102,6 +102,20 @@ void ui_refresh(void)
   }
 }]]
 
+  it('can change the language of the parser', function()
```

Comment:
- change to "active"
- the test is explicitly testing `get_parser` so it is really helpful to explicitly say that in the test name

```suggestion
  it('get_parser changes the language of the parser', function()
```

Reply:
I always thought that the idea of this test framework was to set a topic and then all phrases start with `it` ("Downloader", it downloads a file, it terminates gracefully) 😄